pkg/resources/validation/feature/feature.go

```diff
@@ -49,6 +49,18 @@ func (v *Validator) Admit(request *admission.Request) (*admissionv1.AdmissionRes
 		return nil, err
 	}
 
+	if oldFeature.Name == "multi-cluster-management" && !isValidMCMFeatureValue(oldFeature.Status.Default, oldFeature.Spec.Value, newFeature.Spec.Value) {
+		return &admissionv1.AdmissionResponse{
+			Result: &metav1.Status{
+				Status:  "Failure",
+				Message: "mcm feature flag cannot be changed from true to false",
+				Reason:  metav1.StatusReasonInvalid,
+				Code:    http.StatusBadRequest,
+			},
+			Allowed: false,
+		}, nil
+	}
+
 	if !isValidFeatureValue(newFeature.Status.LockedValue, oldFeature.Spec.Value, newFeature.Spec.Value) {
```

Comment:
Can you elaborate more on why this check (preventing a locked value from changing) does not suit your use case? I.e., can harvester simply lock the mcm feature with the `true` value to meet this use case?

Reply:
From this [comment](https://suse.slack.com/archives/C02CX064EBX/p1680123611416709?thread_ts=1679454772.476669&cid=C02CX064EBX), I think changing the `multi-cluster-management` feature from `true` to `false` is an invalid operation. Also, users can still change `lockedValue` by editing YAML, so it's better to use a webhook to deny the operation.

I also added test cases. Thanks.